bin/license_scout

```diff
@@ -38,6 +38,31 @@ collector = LicenseScout::Collector.new(project_name, project_dir, output_dir, o
 collector.run
 report = collector.issue_report
 
-puts report
+unless report.empty?
+  puts report
 
-exit 2 if !report.empty?
+  puts <<-EXPLANATION
+
+How to fix this depends on what information license_scout was unable to
+determine:
+
+* If the package is missing license information, that means license_scout was
+  unable to determine which license the package was released under. Depending
+  on the package manager, this is usually specified in the package's metadata,
+  for example, in the gemspec file for rubygems or in the package.json for npm.
+  If you know which license a package was released under, MIT for example, you
+  can add an override in license_scout's overrides.rb file in the section for
+  the appropriate package manager like this:
+    ["package-name", "MIT", nil]
+
+* If the package is missing the license file, the means license_scout could not
+  find the license text is any of the locations that typically exists, for
+  example, in a file named LICENSE in the root of the package. If you know that
+  the package was released under one of the common software licenses,
+  MIT for example, you can add an override in license_scout's overrides.rb file
+  in the section for the appropriate package manager like this:
+    ["package-name", nil, [canonical("MIT")]]
```

Comment:
It's preferred that if a package has a license file or text somewhere, for example if there's a LICENSE file in git but not in the package (e.g., not in the `files` gemspec data), that we collect the license the project publishes. Some of our customers and partners will actually check these files for modifications. `canonical` should only be used when a license is specified in metadata but not included in the project's repo/docs/etc.

Reply:
That's great info, I'll rewrite that section to include that and ping you afterwards.